core/services/ardupilot_manager/mavlink_comm/MavlinkComm.py

```diff
@@ -0,0 +1,53 @@
+import json
+from typing import Any, Dict, Optional
+
+import requests
+
+from exceptions import MavlinkMessageReceiveFail, MavlinkMessageSendFail
+
+
+class MavlinkMessenger:
+    def __init__(self) -> None:
+        self.system_id = 255
```

Comment:
Humnnn fair. What should we use them? Anything besides 1 and 255?

Reply:
You should use `MAV_SYSTEM_ID` and maybe `MAV_COMPONENT_ID_ONBOARD_COMPUTER4`.
This variables are now available in the environment system since https://github.com/bluerobotics/companion-docker/pull/421

Reply:
Good